agents/src/transcription.ts

```diff
@@ -14,101 +15,108 @@ export interface TranscriptionForwarder {
   currentCharacterIndex: number;
 }
 
+export enum TranscriptionType {
```

Comment:
`ForwardingType` would be a clearer name. I'd also consider making the options `Transcription` and `Chat` and the setting an array, rather than a third enum value. then you can simplify the forwarding checks  in the loop to

```
if (forwarding_types.contains('transcription')) {
...
}
if (forwarding_types.contains('chat')) {
...
}
```